presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -278,6 +278,20 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testInsertForDefaultColumn()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.default_table (a bigint, b bigint default 40)");
```

Comment:
add this test to abstract smoke tests

Reply:
but the problem in moving it to abstract test is that not all connectors support adding a default value for its column

Reply:
I think it's safe to add it in the `AbstractTestIntegrationSmokeTest` and override it for connectors that don't support this.

some day we will have abstract smoke test for JDBC connectors only.